app/vmalert/web.qtpl

```diff
@@ -30,313 +91,282 @@
-                                            <span><a target="_blank" href="{%s prefix+r.WebLink() %}">Details</a></span>
-                                        </div>
-                                        <div class="col-12">
-                                            <code><pre>{%s r.Query %}</pre></code>
-                                        </div>
-                                        <div class="col-12 mb-2">
-                                            {% if len(r.Labels) > 0 %} <b>Labels:</b>{% endif %}
-                                            {% for k, v := range r.Labels %}
-                                                    <span class="ms-1 badge bg-primary label">{%s k %}={%s v %}</span>
-                                            {% endfor %}
-                                        </div>
-                                        {% if r.LastError != "" %}
-                                        <div class="col-12">
-                                            <b>Error:</b>
-                                            <div class="error-cell">
-                                            {%s r.LastError %}
+            {% for _, g := range groups %}
+                <div id="group-{%s g.ID %}" class="d-flex w-100 border-0 flex-column group-items{% if g.Unhealthy > 0 %} alert-danger{% endif %}">
+                    <span class="d-flex justify-content-between">
+                        <a href="#group-{%s g.ID %}">{%s g.Name %}{% if g.Type != "prometheus" %} ({%s g.Type %}){% endif %} (every {%f.0 g.Interval %}s) #</a>
+                        <span
+                            class="flex-grow-1 d-flex justify-content-end"
+                            role="button"
+                            data-bs-toggle="collapse"
+                            data-bs-target="#sub-{%s g.ID %}"
+                        >
+                            <span class="d-flex gap-2">
+                                {% if g.Unhealthy > 0 %}<span class="badge bg-danger" title="Number of rules with status Error">{%d g.Unhealthy %}</span> {% endif %}
+                                {% if g.NoMatch > 0 %}<span class="badge bg-warning" title="Number of rules with status NoMatch">{%d g.NoMatch %}</span> {% endif %}
+                                <span class="badge bg-success" title="Number of rules with status Ok">{%d g.Healthy %}</span>
+                            </span>
+                        </span>
+                    </span>
+                    <span>
+                        <span
+                            class="fs-6 text-start w-100 fw-lighter"
+                            role="button"
+                            data-bs-toggle="collapse"
+                            data-bs-target="#sub-{%s g.ID %}"
+                        >{%s g.File %}</span>
+                        {% if len(g.Params) > 0 %}
+                            <span class="fs-6 text-start w-100 fw-lighter">Extra params
```

Comment:
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/4cc04778-5ec5-4090-a6fc-aa78b3f66f83" />

`Extra params` and `Extra headers` better have their own divs, below are previous:
<img width="1504" alt="image" src="https://github.com/user-attachments/assets/15acf632-145b-4fda-a820-a9f642060c1a" />

Reply:
updated this

Reply:
extra params and extra headers seems distorted right now:
<img width="756" alt="image" src="https://github.com/user-attachments/assets/31924377-b4a5-42e0-b4dd-b4e63859bf6a" />
<img width="623" alt="image" src="https://github.com/user-attachments/assets/d25cbfa7-9e79-40f1-8cf7-e159396efc65" />